File: backend/VirtualExcursion.DAL/models/Tag.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VirtualExcursion.DAL.models
{
    public class Tag
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Slug { get; set; } = string.Empty;
        public string? Description { get; set; }
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;

        // Navigation
        public ICollection<SceneTag> SceneTags { get; set; } = new List<SceneTag>();
    }
}

File: frontend/src/entities/tags/types/tag.ts
export interface Tag {
    id: number,
    name: string,
    slug?: string,
    description?: string,
    createdAt?: string,
    scenesCount?: number
}

// public int Id { get; set; }
// public string Name { get; set; }
// public string Slug { get; set; }
// public string? Description { get; set; }
// public DateTime CreatedAt { get; set; }
// public int ScenesCount { get; set; }